### PR TITLE
New Package: gimp-lqr-plugin

### PR DESCRIPTION
**Pull request by mvlahos**
Add the liquid resize library, and a gimp plugin that uses it.

### CODE REVIEW

File: srcpkgs/gimp-lqr-plugin/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'gimp-lqr-plugin'
+pkgname=gimp-lqr-plugin
+version=0.7.2
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-configure
+hostmakedepends="automake intltool pkg-config"
+makedepends="gimp-devel glib-devel gettext-devel liblqr-devel"
+#depends="gimp-python"
```
Comment (maxice8) [inline srcpkgs/gimp-lqr-plugin/template]: remove or uncomment

Comment (mvlahos) [reply srcpkgs/gimp-lqr-plugin/template]: Will do.

File: srcpkgs/gimp-lqr-plugin/template [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'gimp-lqr-plugin'
+pkgname=gimp-lqr-plugin
+version=0.7.2
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (Hoshpak) [inline srcpkgs/gimp-lqr-plugin/template]: This is already the default, please remove that line.

File: srcpkgs/liblqr/template [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'liblqr'
+pkgname=liblqr
+version=0.4.2
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (Hoshpak) [inline srcpkgs/liblqr/template]: same here.

File: srcpkgs/gimp-lqr-plugin/template (hunk 2 above)
Comment (mvlahos) [reply srcpkgs/gimp-lqr-plugin/template]: Will do.

File: srcpkgs/liblqr/template (hunk 3 above)
Comment (mvlahos) [reply srcpkgs/liblqr/template]: Will do as well.

**Comment by mvlahos** [comment]
I have rebased on master to fix the common/shlibs issue, and made the requested changes to the templates. 

Please let me know if there is anything else :)

**Pull request merged**